third-party/realdds/src/dds-device-impl.h

```diff
@@ -79,6 +77,12 @@ class dds_device::impl
     void create_control_writer();
     bool init();
 
+    // notification handlers
+    typedef std::map< std::string, void ( dds_device::impl::* )( nlohmann::json const & ) > notification_handlers;
+    static notification_handlers const _notification_handlers;
```

Comment:
Why static?

Reply:
It's not per device, really. It's just a map of id-to-handler. Good enough for now... I even considered making it static inside the `handle_notification` function.